packages/vm/test/api/runBlock.spec.ts

```diff
@@ -58,10 +58,10 @@ const common = new Common({ chain: Mainnet, hardfork: Hardfork.Berlin })
 describe('runBlock() -> successful API parameter usage', async () => {
   async function simpleRun(vm: VM) {
     const common = new Common({ chain: Mainnet, hardfork: Hardfork.London })
-    const genesisRlp = hexToBytes(testData.genesisRLP as PrefixedHexString)
+    const genesisRlp = hexToBytes(testData.default.genesisRLP as PrefixedHexString)
     const genesis = createBlockFromRLPSerializedBlock(genesisRlp, { common })
 
-    const blockRlp = hexToBytes(testData.blocks[0].rlp as PrefixedHexString)
+    const blockRlp = hexToBytes(testData.default.blocks[0].rlp as PrefixedHexString)
```

Comment:
Are all these now commonly applied `default` accesses future proof? (so: not only work in our specific current tooling setup)

Reply:
@acolytec3 maybe a final comment on this one?

Reply:
I think it's fine, though honestly not 100% sure. I think it was a knock-on effect of switching our dev setup to `module` in our package.json across the monorepo.

Reply:
Ok, thanks, then let's keep/take it, though I find all these kind of things honestly a bit spooky